lib/server/services/lhcPeriod/LhcPeriodStatisticsService.js

```diff
@@ -211,31 +150,52 @@ class LhcPeriodStatisticsService {
             }
         }
 
-        queryBuilder.includeAttribute({
-            query: DISTINCT_ENERGIES_IN_STATISTICS_SUBQUERY,
-            alias: 'distinctEnergies',
-        });
-        queryBuilder.includeAttribute({
-            query: PDP_BEAM_TYPES_IN_STATISTICS_SUBQUERY,
-            alias: 'beamType',
-        });
-        queryBuilder.includeAttribute({
-            query: DATA_PASSES_COUNT_SUBQUERY,
-            alias: 'dataPassesCount',
-        });
-
-        queryBuilder.includeAttribute({
-            query: RUNS_COUNT_SUBQUERY,
-            alias: 'runsCount',
-        });
-
         const { count, rows } = await LhcPeriodStatisticsRepository.findAndCountAll(queryBuilder);
 
         return {
-            count,
+            count: count.length,
```

Comment:
Please put a comment on why you need to do that

Reply:
added